api/wallet_test.go

```diff
@@ -1437,3 +1438,82 @@ func TestWalletChangePassword(t *testing.T) {
 		t.Error("wallet is not unlocked")
 	}
 }
+
+// TestWalletSiacoins tests the /wallet/siacoins endpoint, including sending
+// to multiple addresses.
```

Comment:
This test only checks that send-to-many works, does not check that single sends work. Granted, we use single sends all over the place for API testing I don't think it's something that strictly needs a separate test, but docstring is misleading.